apps/pie-storybook/package.json

```diff
@@ -34,6 +34,7 @@
     "dompurify": "3.0.11"
   },
   "devDependencies": {
+    "@justeattakeaway/pie-icons-configs": "4.5.1",
```

Comment:
I'm not entirely sure how this works with storybook, but these are used inside the story, so should this go in `dependencies`?

Reply:
I was unsure as well. its used to build the story so I thought dev deps but open to suggestions

Reply:
I guess if it works it's fine for now 🤷🏻‍♂️